### PR TITLE
feat: pass custom type for any asssets with phoenix bodytype

### PR DESCRIPTION
**Pull request by rk132**

### CODE REVIEW

File: schemas/gltf-asset.schema.json [hunk 1]
```diff
@@ -13,6 +13,14 @@
         "type": "Asset type should be a string."
       }
     },
+    "bodyType": {
+      "description": "The bodyType of asset.",
+      "type": "string",
+      "default": "custom",
```
Comment (Olaf-Wolf3D) [inline schemas/gltf-asset.schema.json:19]: It is a bit confusing that the default value "custom" for bodyType does not trigger the custom asset schema (assetCustom), but the regular asset types. Which schema should the default value trigger?

File: schemas/gltf-asset.schema.json [hunk 2]
```diff
@@ -13,6 +13,14 @@
...
+      "errorMessage": {
+        "type": "Bodytype should be a string."
```
Comment (Olaf-Wolf3D) [inline schemas/gltf-asset.schema.json:21]: Please add a space between Body and type, the it matches with assetType error message.

**Review by Olaf-Wolf3D — CHANGES_REQUESTED** [review]

**Review by Olaf-Wolf3D — APPROVED** [review]
If this is tested and made sure that regular assets still upload in studio 1.0 asset manager, this can be merged.

**Comment by MaxAndreassenRPM** [comment]
Merged as per this thread -> https://readyplayerme.slack.com/archives/C042V5Q9HK9/p1729165561063299?thread_ts=1728564057.961929&cid=C042V5Q9HK9

**Pull request merged**